docs/README.md

```diff
@@ -1,3 +1,38 @@
-# Order Profile
+📢 Use this project, [contribute](https://github.com/{OrganizationName}/{AppName}) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
+
+# ORDER PROFILE
+
+⚠️ **This is an ongoing, unsupported, unfinished and undocumented project. We do not guarantee any results after installation.** ⚠️
+
+<!-- DOCS-IGNORE:start -->
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+<!-- DOCS-IGNORE:end -->
 
 Centralizes all profile related requests to the Checkout API
```

Comment:
The Order Profile app is responsible for storing all user profiles that perform requests to the [VTEX Checkout API](https://developers.vtex.com/vtex-developer-docs/reference/checkout-api-overview).